react-markdown/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+	"module": "commonjs",
+	"target": "es6",
+	"noImplicitAny": true,
+	"strictNullChecks": true,
+	"baseUrl": "../",
+	"typeRoots": [
+	  "../"
+	],
+	"types": [],
+  "jsx" : "react",
+	"noEmit": true,
+	"forceConsistentCasingInFileNames": true
+  },
+  "files": [
```

Comment:
These should be "index.d.ts" and "react-markdown-tests.tsx". Make sure to test with `tsc`.